src/en/articles/landmarks/index.njk

```diff
@@ -1,136 +1,236 @@
 ---
-title: "Landmarks ARIA"
-abstract: "How to improve page navigation by adding semantics to content that makes it easier to find the major regions of the page"
+title: "Structuring a page with ARIA landmarks"
+abstract: "Setting up ARIA landmarks."
 titleBeforeTag: true
 date: "2018-01-15"
+updateDate: "2025-05-30"
 tags:
   - component
+css:
+  - style.css
 ---
 
-    <h2>Introduction</h2>
-    <p>In this example, we'll see how to improve page navigation by adding semantics to content that makes it easier to find the major regions of the page.</p>
-    <h2> Theory</h2>
-    <p><abbr>HTML5</abbr> introduces new tags, including semantic tags (<code>main</code> , <code>aside</code> , <code>footer</code> , and <code>header</code> , ...).</p>
-    <p>These tags should be used alone without the corresponding <abbr>ARIA</abbr> roles because their support is good in the vast majority of browsers and assistive devices.</p>
-
-    For a page in <abbr>HTML5</abbr>:
-
-  <pre><code class="html">
-  &lt;head&gt;
-    …
-    &lt;title&gt;Titre de la page&lt;/title&gt;
-    …
-  &lt;/head&gt;
-  &lt;body&gt;
-    &lt;header&gt;
-      &lt;h1&gt;…&lt;/h1&gt;
-      &lt;nav&gt;…navigation…&lt;/nav&gt;
-      &lt;p&gt;page header&nbsp;: logo, baseline…&lt;/p&gt;
-      …
-    &lt;/header&gt;
-    &lt;main&gt;
-       &lt;nav aria-label="Breadcrumb"&gt;[…]&lt;/nav&gt;
-       &lt;section&gt;
-         &lt;header&gt;
-           &lt;h1&gt;News&lt;/h1&gt;
-           &lt;p&gt;June 21th, 2018 - Music festival&lt;/p&gt;
-         &lt;/header&gt;
-         &lt;p&gt;section's content&lt;/p&gt;
-         &lt;footer&gt;
-           &lt;p&gt;section footer content &nbsp;: copyright…&lt;/p&gt;
-           …
-         &lt;/footer&gt;
-      &lt;/section&gt;
-      &lt;article&gt;
-      […]
-      &lt;/article&gt;
+<h2>Introduction</h2>
 
-      […]
+<p>In this article, we'll look at how to <span class="fw-bold">improve web page navigation</span> by programmatically structuring content using semantic <abbr>HTML</abbr> elements.</p>
+<p>This structure will enable users who navigate with a <span class="fw-bold">technical aid</span> (e.g., a screen reader) or only with the <span class="fw-bold">keyboard</span>, to be able to visualize the page content via their assistance tools, without having to browse through it. This saves a considerable amount of time and energy.</p>
```

Comment:
```suggestion
<p>This structure will enable users who navigate with <span class="fw-bold">assistive technologies</span> (e.g., a screen reader) or only with the <span class="fw-bold">keyboard</span>, to better understand the page content through their tools, without having to browse through it.</p>
```

Reply:
Just a not mandatory suggestion to keep 'assistive technologies' like in the rest of the article.